### PR TITLE
Add test coverage to the cmd package

### LINKED ISSUE #78: Add cobra command and subcommand testing
In order to improve test coverage and catch inconsistencies between commands we need to add cobra sub command testing. Here is a link provided by @komish that we can use as a starting point: https://g14a.dev/posts/Testing-Cobra-Subcommands/

### PR DESCRIPTION
**Pull request by bcrochet**
Signed-off-by: Brad P. Crochet <brad@redhat.com>

## Description of PR
Add test coverage to cmd package.

Fixes #78

## Changes (required)

* Add tests for check command
* Add tests for upload command

## Checklist (required)
- [X] I have reviewed and followed the [contribution guidelines](https://github.com/opdev/opcap/blob/main/docs/contribution.md)
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation if needed
- [X] I have checked that my changes pass all tests

### CODE REVIEW

**Review by skattoju — APPROVED** [review]
/lgtm

File: cmd/upload.go [hunk 1]
```diff
@@ -116,23 +117,55 @@ func uploadRunE(cmd *cobra.Command, args []string) error {
...
+
+type minioClient interface {
+	BucketExists(ctx context.Context, bucket string) (bool, error)
+	MakeBucket(ctx context.Context, bucket string, opts minio.MakeBucketOptions) error
+	FPutObject(ctx context.Context, bucket, path, file string, opts minio.PutObjectOptions) (minio.UploadInfo, error)
+}
+
+func loadAudits(ctx context.Context, afs afero.Fs, filename string) ([]Audit, error) {
+	// Initialize audits array with a capacity of 10 and a length of 0
+	audits := make([]Audit, 0, 10)
```
Comment (acmenezes) [inline cmd/upload.go:137]: Are we only catching 10 audits out of a file here? I may have misread something.

**Review by acmenezes — CHANGES_REQUESTED** [review]
Not sure why the audits array has only 10 as its length.

File: cmd/upload.go (hunk 1 above)
Comment (bcrochet) [reply cmd/upload.go:137]: No. This is just an initial allocation.

Comment (bcrochet) [reply cmd/upload.go:137]: I don't see anything that would inform us of a better number.

**Review by yashoza19 — APPROVED** [review]
/approve

**Pull request merged**